images/airflow-ansible/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=quay.io/cloud-bulldozer/airflow:2.3.2
+ARG BASE_IMAGE=quay.io/cloud-bulldozer/airflow:latest
```

Comment:
I realize this is our own repo, but could tracking latest be a bad idea?

Reply:
this variable is actually overwritten by podman, so we can ignore it

Reply:
Just removed the default value to avoid confusion